controllers/controllers.js

```diff
@@ -166,4 +170,24 @@ exports.getUsername = (req, res, next) => {
     .catch(err => {
         next(err);
     })
+}
+
+exports.delComment = (req, res, next) => {
+    const id = req.params["commId"];
```

Comment:
This should be extracted to the model, the controller should handle the request response and the model should handle all the data queries and formatting